tests/virt_autotest/kubevirt_tests_server.pm

```diff
@@ -264,33 +296,159 @@ sub deploy_kubevirt_manifests {
     record_info('List local storage', script_output('ls /mnt/local-storage -R -l'));
     record_info('Check the loop device "loop0"', script_output('losetup /dev/loop0'));
     record_info('Check all loop devices', script_output('losetup -l -a'));
+
+    $self->apply_test_config();
 }
 
-sub run_virt_tests {
+sub setup_longhorn_csi {
     my $self = shift;
-    my $test_conf;
-    my @kubevirt_tests;
-    my $test_suite_config = '/usr/share/kube-virt/manifests/testing/default-config.json';
 
-    record_info('Run kubevirt tests', '');
-    transactional::enter_trup_shell(global_options => '--drop-if-no-change') if (is_transactional);
-
-    record_info('Set local storage class', '');
-    assert_script_run("sed -i '/storageClassLocal/s/local/local-path/' $test_suite_config");
+    record_info('Install Longhorn CSI', '');
+
+    # Install Longhorn CSI
+    assert_script_run("kubectl apply -f https://raw.githubusercontent.com/longhorn/longhorn/v1.4.x/deploy/longhorn.yaml");
+
+    # Ensure successful Longhorn deployment
+    my @deployments = split(/\n/, script_output("kubectl get --no-headers deployments -n longhorn-system -o custom-columns=:.metadata.name"));
+    assert_script_run("kubectl rollout status deployment --timeout=20m -n longhorn-system $_") foreach (@deployments);
+    my @daemonsets = split(/\n/, script_output("kubectl get --no-headers daemonsets -n longhorn-system -o custom-columns=:.metadata.name"));
+    assert_script_run("kubectl rollout status daemonset --timeout=20m -n longhorn-system $_") foreach (@daemonsets);
```

Comment:
So does here?

Reply:
Refer to https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/16708#discussion_r1236945754